### ISSUE TITLE
TextButton - resolve contrast violation in disabled state

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/B01LER-Kitchen#660]
#### Step-by-Step Guide to reproduce

1. Open TextButton in Storybook
2. Set disabled to true
3. See Contrast Violation in Accessibility Add-on

----
#### Bug Description / False Behaviour

The disabled TextButton has a contrast violation.

----
#### Correct Behaviour

The disabled TextButton does not have a contrast violation.

----
#### Testing

_Check existing unit tests of the component(s) and adjust the tests to prevent this bug from occurring again in the future, if you see a way._

----
#### Background
- According to https://www.w3.org/TR/WCAG21/#contrast-minimum inactive components do not need to comply to the contrast minimum.
- A quick investigation showed, that the add-on just doesn't know that the element is disabled. If we change the span to a button element with disabled attribute, the violation will disappear.
- We decided to have a span to make the button work in safari.
- We might be able to use aria-disabled to solve the issue.

**Comment by thrbnhrtmnn** [comment]
After merge this issue can be closed as complete. No open feedback from the design review.
